app/controllers/users_controller.rb

```diff
@@ -14,4 +15,8 @@ def edit
         .by_start_at
     )
   end
+
+  def activities
```

Comment:
Are you going to merge this with the `ActivitiesController#index` action as we discussed? It feels wrong hanging off here.